tests/Feature/HeaderTestCase.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GacelaTest\Feature;
+
+use PHPUnit\Framework\TestCase;
+
+include_once __DIR__ . '/header.php';
+
+abstract class HeaderTestCase extends TestCase
+{
+    /**
+     * @runInSeparateProcess
```

Comment:
This doesn't work if we use a trait right? In other words, we are skipping the composition over inheritance due to a technical limitation. @Chemaclass

Reply:
That's 100% correct. This works on a class level, but not on trait level